sphinx_docs/conf.py

```diff
@@ -103,8 +103,7 @@
     if "GIT_ORIGIN_URL" not in os.environ:
         base_url = ""  # If this is non-empty, sphinx will make it clickable.
     else:
-        owner_name = os.path.splitext(
-            os.environ.get("GIT_ORIGIN_URL", "").split(":")[1]
-        )[0]
+        owner_name = os.environ["GIT_ORIGIN_URL"].split(":")[1].rsplit(".", 1)[0]
+        owner_name = "/".join(owner_name.split("/")[-2:])
```

Comment:
The logic all looks good to me, so I think this PR can merge as is.

I do think there's opportunity to make this all a little nicer if you'd like to do so in a FF.

First, I see`if "GIT_ORIGIN_URL" not in os.environ:` and then `os.environ["GIT_ORIGIN_URL"]`. Also, the `if not, else` seems like an unnecessary `not` to me (just switch the order).
```py
git_origin_url = os.environ.get("GIT_ORIGIN_URL", None)
if git_origin_url:
    owner_name = git_origin_url.split(...)...
    ...
else:
    base_url = ""  # If this is non-empty, sphinx will make it clickable.
```

Next, I prefer to see this specific string manipulation logic in a helper function 1) so it is isolated and 2) so it is easier to document with examples:

```py
def owner_name_from(git_origin_url):
    """
    Extract the owner name from a git origin URL.

    The git origin URL might be in ``git+ssh`` form, or ``http`` form.

    Examples:
        >>> owner_name_from(<git+ssh example>)
        "<answer>"
        >>> owner_name_from(<http example>)
        "<answer>"
```

Honestly having that in this PR would be really helpful since I don't really have the full context of what `owner_name` needs to be. It would be easier to review the logic seeing the examples documented.

Reply:
I approved but then Lewis said he would rather do this now than as a FF so I'm going to hold off on doing the merge.